reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/processor/source/invokes/BaseInvokesProcessor.kt

```diff
@@ -23,5 +26,11 @@ abstract class BaseInvokesProcessor<Output : Any>(override val binding: BindingC
 
     protected abstract fun isValidExpression(expression: KtReferenceExpression): Boolean
 
-    override fun shouldRunOn(element: KtElement) = (element as? KtReferenceExpression)?.let { isValidExpression(it) } ?: false
+    private fun isValidExpressionWithLog(expression: KtReferenceExpression): Boolean {
```

Comment:
kinda strange to add 'withLog' to its name, what's the point? we have many functions that log something

Reply:
Since we would like to override just isValidExpression function and use logging always, it is necessary for debugging and searching weak places

Reply:
well okay